session-replacement/src/main/java/com/amadeus/session/ErrorTracker.java

```diff
@@ -0,0 +1,58 @@
+package com.amadeus.session;
+
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+/**
+ * The class is used to track the number of item during a time.
+ * input parameter
```

Comment:
*  @param delay 
                  Time in milliseconds while items are kept
*  @param max
                   If the number of items is more than the max value the Tracker is considered in error